src/main/R/drtDemandAnalysis/VIA-data/KEXI-plot-time-series-VIA.R

```diff
@@ -137,6 +150,9 @@ for(dataset in datasets) {
     geom_line(mapping=aes(x=date, y=n), col="#69b3a2") +
     geom_area(mapping=aes(x=date, y=n), fill="#69b3a2", alpha=0.5) +
     labs(x="Tag", y="Fahrten", title="Zeitverlauf der Fahrten pro Tag (VIA)") +
+    theme(plot.title = element_text(hjust=0.5, size=3.5, face="bold"), axis.text.x = element_text(size=2),
```

Comment:
This is a comparison of how the plots looked like before vs now

![grafik](https://github.com/matsim-scenarios/matsim-kelheim/assets/13538764/c3a39956-a3ce-4956-8ad0-7810295269b9)


Interestingly, the line seems to appear bolder now, but i don't see the corresponding change in the code...
Even though I think it looked better, before, I think we can merge..